tests/fixture/e2e/helpers.go

```diff
@@ -195,12 +198,25 @@ func WithSuggestedGasPrice(ethClient ethclient.Client) common.Option {
 
 // Verify that a new node can bootstrap into the network.
 func CheckBootstrapIsPossible(network testnet.Network) {
+	require := require.New(ginkgo.GinkgoT())
+
 	if len(os.Getenv(SkipBootstrapChecksEnvName)) > 0 {
 		tests.Outf("{{yellow}}Skipping bootstrap check due to the %s env var being set", SkipBootstrapChecksEnvName)
 		return
 	}
 	ginkgo.By("checking if bootstrap is possible with the current network state")
-	node := AddEphemeralNode(network, testnet.FlagsMap{})
+
+	// Call network.AddEphemeralNode instead of AddEphemeralNode for compatibility
+	// with ginkgo.DeferCleanup. It's not possible to call DeferCleanup from
+	// within a function called by DeferCleanup.
+	node, err := network.AddEphemeralNode(ginkgo.GinkgoWriter, testnet.FlagsMap{})
+	defer func() {
+		if node != nil {
+			tests.Outf("Shutting down ephemeral node %s\n", node.GetID())
+			require.NoError(node.Stop())
+		}
+	}()
+	require.NoError(err)
```

Comment:
It feels weird to me to be shutting down the node if `err != nil`, 
```suggestion
	require.NoError(err)
	defer func() {
		tests.Outf("Shutting down ephemeral node %s\n", node.GetID())
		require.NoError(node.Stop())
	}()
```
Can we avoid the `nil` check if we move the `defer` to after the error check?